tox.ini

```diff
@@ -4,8 +4,8 @@ minversion = 3.18.0
 envlist =
     lint
     packaging
-    py{36,37,38,39,310}
-    py{36,37,38,39,310}-{devel}
+    py{38,39,310}
```

Comment:
I'm afraid you missed lines 60 and 61 in this file. They also reference `py36` and `py37`.

Reply:
Thanks!
Will review and fix on next PR